common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/BlockTransferMessage.java

```diff
@@ -35,6 +35,9 @@
  *     shuffle service. It returns a StreamHandle.
  *   - UploadBlock is only handled by the NettyBlockTransferService.
  *   - RegisterExecutor is only handled by the external shuffle service.
+ *   - StreamHandle is only handled by the OneForOneBlockFetcher.
+ *   - RegisterDriver is only handled by the MesosExternalShuffleService.
+ *   - HearBeat is only handled by the MesosExternalShuffleService.
```

Comment:
HeartBeat. (This could also be an HTML list.) I am not sure these docs match the others. They're describing what's handled by the Netty service for external shuffle block handler, not just a class where they're used.